pkgs/development/python-modules/pandas/default.nix

```diff
@@ -49,7 +49,8 @@ buildPythonPackage rec {
 
   nativeCheckInputs = [
     glibcLocales
-    hypothesis
+    # hypothesis indirectly depends on pandas to build its documentation
+    (hypothesis.override { enableDocumentation = false; })
```

Comment:
Not sure if we should move this to python-packages

Reply:
Presumably we still want the top-level `hypothesis` to have documentation?

Reply:
I mean if we should move the overwrite to the attr set passed to pandas callPackage.